tests/e2e/package.json

```diff
@@ -1,34 +1,34 @@
 {
-  "scripts": {
-    "pretest:e2e": "./bin/local-env/env-check.sh && ./bin/test-docker-connectivity",
-    "test:e2e": "cross-env NODE_OPTIONS=--experimental-abortcontroller WP_BASE_URL=http://localhost:9002 ./script.js",
```

Comment:
Ideally this should happen in a separate issue, but I was going to propose the same, so I'm glad we're doing it.